asistente_ladm_col/gui/reports.py

```diff
@@ -376,6 +376,6 @@ def remove_report_dependency(self):
 
         if os.path.exists(base_path):
             self.qgis_utils.message_with_duration_emitted.emit(
-                QCoreApplication.translate("ReportGenerator", "It wasn't possible to remove the dependency folder. You need to remove this folder yourself to generate reports: <a href='file://{path}'>{path}</a>").format(path=base_path),
+                QCoreApplication.translate("ReportGenerator", "It wasn't possible to remove the dependency folder. You need to remove this folder yourself to generate reports: <a href='file:///{path}'>{path}</a>").format(path=base_path),
```

Comment:
Missing function call.

Reply:
Fix in: b0489ea69759bd71d7bb4ed01707be32d456f33a